docs/streams/developer-guide/config-streams.html

```diff
@@ -97,6 +97,7 @@
             <ul>
               <li><a class="reference internal" href="#naming" id="id17">Naming</a></li>
               <li><a class="reference internal" href="#default-values" id="id18">Default Values</a></li>
+              <li><a class="reference internal" href="#parameters-controlled-by-kafka-streams" id="id26">Default Values</a></li>
```

Comment:
I suppose this should be
```suggestion
              <li><a class="reference internal" href="#parameters-controlled-by-kafka-streams" id="id26">Parameters controlled by Kafka Streams</a></li>
```

Reply:
Thanks. Great catch. C&p error.